docs/content/contribute/run-hypershift-operator-locally.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Run the hypershift-operator locally
+---
+
+## Run the HyperShift Operator locally
+
+To run the HyperShift Operator locally, follow these steps:
+
+1. Ensure that the `KUBECONFIG` environment variable is set to a management cluster where HyperShift has not been installed yet.
+
+  ```shell linenums="1"
+   export KUBECONFIG="/path/to/your/kubeconfig"
+  ```
+
+2. Build HyperShift.
+
+!!! note     
+ 
+    `requires go v1.22+
+
+```shell linenums="1"
+  make build
+```
+
+3. Set the necessary environment variables
+
+  ```shell linenums="1"
+    export HYPERSHIFT_REGION="your-region"
+    export HYPERSHIFT_BUCKET_NAME="your-bucket"
+  ```
+
+!!! note 
+
+    `Consider setting HYPERSHIFT_REGION and HYPERSHIFT_BUCKET_NAME in your shell init script (e.g., $HOME/.bashrc).
+
+4. Install HyperShift in development mode which causes the operator deployment to be deployment scaled to zero so that it doesn't conflict with your local operator process (see [Prerequisites](../getting-started.md#prerequisites)):
+  
+  ```shell linenums="1"
+    HYPERSHIFT_REGION=us-east-2 HYPERSHIFT_BUCKET_NAME=my-bucket make hypershift-install-aws
```

Comment:
why do I need to export the variables and then pass them again?

Reply:
Also can we mention you can set them but there's defaults and `make whatever` should just work without exporting or setting anything

Reply:
You are exporting the vars because in the script the default is us-east-1 and bucket is hypershift-ci-oidc. So you'll have to export the correct var if default is not what is needed